backend/danswer/natural_language_processing/utils.py

```diff
@@ -111,18 +111,12 @@ def _check_tokenizer_cache(tokenizer_name: str) -> BaseTokenizer:
     return _TOKENIZER_CACHE[tokenizer_name]
 
 
```

Comment:
This now just uses the default tokenizer no matter what.
Extra character handling is done elsewhere

Reply:
there's some irony here and I love it lmao